client/src/pages/Contest/Contest.tsx

```diff
@@ -0,0 +1,204 @@
+import AuthHeader from '../../components/AuthHeader/AuthHeader';
+import ProfilePic from '../../Images/profilePic.png';
+import { Submission } from '../../interface/User';
+import { getContestById } from '../../helpers/APICalls/contest';
+
+import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
+import { Link, useParams } from 'react-router-dom';
+
+import { withStyles } from '@material-ui/core/styles';
+
+const GlobalCss = withStyles({
+  // @global is handled by jss-plugin-global.
+  '@global': {
+    // You should target [class*="MuiButton-root"] instead if you nest themes.
+    '.MuiToolbar-root': {
+      padding: 0,
+    },
+    '.MuiToolbar-regular': {
+      minHeight: 0,
+    },
+    'li.MuiImageListItem-root': {
+      height: '250px !important',
+      overflowY: 'hidden',
+    },
+    '.MuiImageListItemBar-root': {
+      background: 'none',
+    },
+    'ul.li.MuiImageListItem-item': {
+      padding: 0,
+      height: '250px !important',
+    },
+    '.MuiImageListItemBar-titleWrap': {
+      color: '#333',
+      marginTop: '12px',
+    },
+    '.MuiImageListItemBar-title': {
+      fontSize: '15px',
+      fontWeight: 'bolder',
+      textDecoration: 'underline',
+    },
+    '.MuiImageListItem-imgFullHeight': {
+      opacity: '0.8',
+    },
+  },
+})(() => null);
+
+export default function Contest(): JSX.Element {
+  const { id } = useParams() as { id: string };
+  const classes = useStyles();
+  const [value, setValue] = useState(0);
+  //const [contestSubmissions, setContestSubmissions] = useState<Submission[]>([]);
+  //currently leaving as any, need to change it after getting individual contest from api
+  const [contestSubmissions, setContestSubmissions] = useState<any>([]);
+  const { loggedInUser } = useAuth();
+
+  const newTheme = createMuiTheme({
+    palette: {
+      primary: {
+        main: '#000000',
+      },
+      secondary: {
+        main: '#3d3d3d',
+      },
+    },
+  });
+
+  interface TabPanelProps {
+    direction?: string;
+    value: number;
+    children: React.ReactNode;
+    index: number;
+  }
+
+  useEffect(() => {
+    async function getContestAllSubmissions() {
+      try {
+        const contestAllSubmissions = await getContestById(id);
+        if (contestAllSubmissions) {
+          setContestSubmissions(contestAllSubmissions);
+        }
+      } catch (err) {
+        console.log(err);
+      }
+    }
+    getContestAllSubmissions();
+  }, [id]);
+
+  const handleChange = (event: React.ChangeEvent<Record<string, unknown>>, valueChange: number) => {
+    setValue(valueChange);
+  };
+
+  const Panel = function (props: TabPanelProps) {
+    return (
+      <div role="tabpanel" id={`${props.index}`}>
+        {value === props.index && (
+          <Container>
+            <Typography>{props.children}</Typography>
+          </Container>
+        )}
+      </div>
+    );
+  };
+
+  const capitalize = (s: string) => s && s[0].toUpperCase() + s.slice(1);
```

Comment:
CSS has a text-transform property to capitalize strings, please use that instead :)

Reply:
This is fixed.